packages/x-data-grid/src/models/gridBaseSlots.ts

```diff
@@ -71,6 +71,22 @@ export type CircularProgressProps = {
 
 export type LinearProgressProps = {};
 
+export type InputProps = {
+  ref?: React.Ref<HTMLInputElement>;
+  inputRef?: React.Ref<HTMLInputElement>;
```

Comment:
Since `inputRef` will contain the actual input element, `ref` won't contain one, right?
Should it be `React.Ref<HTMLElement>` then?